extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -112,38 +112,47 @@ export class AzdataTool extends ToolBase {
 	private get azdataInstallAdditionalArgs(): string {
 		return vscode.workspace.getConfiguration(DeploymentConfigurationKey)[azdataPipInstallArgsKey];
 	}
+
+	// Additional Informational messages for various OsTypes.
+	protected additionalInformation: Map<OsType, InformationalMessageType[]> = new Map<OsType, InformationalMessageType[]>([
+		[OsType.linux, []],
```

Comment:
as of now, linux also depends on python and pip3

Reply:
fixed